libparsec/crates/testbed/src/env.rs

```diff
@@ -849,11 +849,17 @@ impl TestbedEnv {
     }
 }
 
+pub enum TestbedTimeToLive {
+    Unlimited,
+    Limited { seconds: u32 },
+}
+
 /// `test_new_testbed` should be called when a test starts (and be followed
 /// by a `test_drop_testbed` call at it end)
 pub async fn test_new_testbed(
     template: &str,
     server_addr: Option<&ParsecAddr>,
+    ttl: TestbedTimeToLive,
```

Comment:
```suggestion
    tttl: TestbedTimeToLive,
```

Should be TTTL for **T**estbed **T**ime **T**o **L**ive :trollface:

Reply:
`ttttl` *T*es*T*bed *T*ime *T*o *L*ive ? ^^